app/controllers/concerns/blacklight/default_component_configuration.rb

```diff
@@ -64,39 +65,26 @@ def add_show_tools_partial name, opts = {}
         end
       end
 
-      ##
       # Add a tool to be displayed for each document in the search results.
-      # @param partial [String] the name of the document partial
-      # @param opts [Hash]
-      # @option opts [Symbol,Proc] :if render this action if the method identified by the symbol or the proc evaluates to true.
-      #                             The proc will receive the action configuration and the document or documents for the action.
-      # @option opts [Symbol,Proc] :unless render this action unless the method identified by the symbol or the proc evaluates to true
-      #                             The proc will receive the action configuration and the document or documents for the action.
-      def add_results_document_tool name, opts = {}
+      # @see #add_nav_action for params
```

Comment:
> I don't think the see looks great actually,

I agree, but again, we can use the YARD macro syntax, remove duplicated documentation text, and maintain a good user experience.

Seeing `@see` in context makes me hate it even more; as a user, I click `for params`, get bounced around the page, and it's unclear what the association between the two locations is.